src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/MethodologyService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Repository.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Publisher.Model.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Services
+{
+    public class MethodologyService : IMethodologyService
+    {
+        private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
+        private readonly IMapper _mapper;
+        private readonly IMethodologyRepository _methodologyRepository;
+
+        public MethodologyService(IPersistenceHelper<ContentDbContext> persistenceHelper,
+            IMapper mapper,
+            IMethodologyRepository methodologyRepository)
+        {
+            _persistenceHelper = persistenceHelper;
+            _mapper = mapper;
+            _methodologyRepository = methodologyRepository;
+        }
+
+        public async Task<Either<ActionResult, MethodologyViewModel>> Get(string slug)
+        {
+            // TODO SOW4 EES-2375 lookup the MethodologyParent by slug when slug is moved to the parent
+            // For now, this does a lookup on the parent via any Methodology with the slug
+            return await _persistenceHelper
+                .CheckEntityExists<Methodology>(
```

Comment:
Nice use of the existence check and hydration!

Reply:
Thanks 🙂